CHANGELOG.md

```diff
@@ -1,12 +1,39 @@
 # App Center SDK for React Native Change Log
 
-## Version 1.10.0 (Under development)
+## Version 1.10.0
+
+### Features
+
+### AppCenterAnalytics
+
+- **[Android]** Add API to specify event persistence priority.
```

Comment:
The two "###AppCenterAnalytics" features you mentioned are only for native Android or native iOS, not for RN. Actually there is no new RN features in this release. Only bug-fixes.